src/reporters/json.ts

```diff
@@ -23,7 +23,8 @@ import { Test, Suite, Spec, TestResult, TestError } from '../test';
 export interface SerializedSuite {
   title: string;
   file: string;
-  location: string,
+  line: (number|undefined);
```

Comment:
Make those required? We will almost always have them...

Reply:
done